packages/time/src/types.ts

```diff
@@ -290,17 +290,19 @@ export type TimeMathType = {
   ) => RelativeTimeRecord;
   /**
    * An absolute time + a relative time gives a new absolute time.
-   *
-   * @template {Timestamp} T
```

Comment:
this was ignored before because it wasn't a `.js` file.

When I made converted the syntax, it narrowed too much all the way to the value literal. That's why made the conditional to return one of two types